integrations/tensorflow/test/tfe2e/math/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+include(iree_test_matrix)
+
+iree_gen_test_matrix()
+
+file(GLOB
+  _GEN_PY_TESTS
+  LIST_DIRECTORIES NO
+  RELATIVE "${CMAKE_CURRENT_BINARY_DIR}/generated"
+  "${CMAKE_CURRENT_BINARY_DIR}/generated/*.py")
```

Comment:
We moved away from CMake's `GLOB` for other IREE tests in https://github.com/google/iree/pull/5063. Maybe this case is different enough for it to make sense.

@GMNGeoffrey

Reply:
Yeah, we want this to be a glob, and the generation script lays things out so that is the right thing to be doing.